docs/data/sdks/android/index.md

```diff
@@ -5,17 +5,22 @@ icon: material/android
 ---
 
 
-This is the official documentation for the Amplitude Analytics Android SDK. This is a legacy SDK. If you haven't already deployed this SDK, we recommend using the [Android Kotlin SDK](../android-kotlin) instead.
+[![Maven Central](https://img.shields.io/maven-central/v/com.amplitude/android-sdk.svg?label=Maven%20Central)](https://search.maven.org/search?q=g:%22com.amplitude%22%20AND%20a:%22android-sdk%22)
+
+This is the official documentation for the Amplitude Analytics Android SDK.
+
+!!!info "Legacy SDK"
```

Comment:
There are a lot of info blocks in a row. I think it would be good to make the legacy notice more pronounced. Brian used a warning block in the RN docs for "Ampli not supported", I think that would be appropriate here. Or maybe we can make a new "admonition" that is more distinct. WDYT? cc @caseyamp 
<img width="281" alt="image" src="https://user-images.githubusercontent.com/5790872/185002873-af765a32-5163-428f-8fd2-adc943c41256.png">

Reply:
changed to attention admonition.